src/controllers/match.ts

```diff
@@ -21,8 +21,6 @@ export const match = async (
 ) => {
   try {
     const jobSeeker = req.user as JobSeeker
-    console.log('JOBSEEKER', jobSeeker)
-
     if (!jobSeeker) return next(new NotFoundError('User not found'))
```

Comment:
I think we can skip this check in all controllers that is behind the `tokenVerify` middleware, because the middleware already handles this for us.